assets/js/components/settings/SettingsAdmin.js

```diff
@@ -80,29 +81,46 @@ export default function SettingsAdmin() {
 		<Row>
 			{ userInputEnabled && (
 				<Cell size={ 12 }>
-					{ isUserInputCompleted && (
-						<Layout
-							title={ __( 'Key metrics', 'google-site-kit' ) }
-							header
-							rounded
-						>
-							<div className="googlesitekit-settings-module googlesitekit-settings-module--active googlesitekit-settings-user-input">
-								<Grid>
+					<Layout
+						title={ __( 'Key metrics', 'google-site-kit' ) }
+						header
+						rounded
+					>
+						<div className="googlesitekit-settings-module googlesitekit-settings-module--active googlesitekit-settings-user-input">
+							<SettingsKeyMetrics />
+							<Grid>
+								{ isUserInputCompleted && (
 									<UserInputPreview
 										goTo={ goTo }
 										noHeader
 										noFooter
 										settingsView
 										showIndividualCTAs
 									/>
-								</Grid>
-							</div>
-						</Layout>
-					) }
-
-					{ isUserInputCompleted === false && (
-						<UserInputSettings isDismissible={ false } rounded />
-					) }
+								) }
+								{ isUserInputCompleted === false && (
+									<div className="googlesitekit-user-input__notification">
```

Comment:
Ok, I see you added a comment for this. Please, create a new issue with your comment and revert this change because we need a more holistic approach to solve that problem since that notification is shown as a banner as well and we need to make sure that we don't show that notification unnecessarily.

Reply:
Hi, the `UserInputSettings` will be removed as part of #6210, and is only displayed in Dashboard (not in Settings). If I revert this change, then there will be multiple titles which is not desired and does not satisfy the ticket. I have updated the `SettingsAdmin` to make proper usage of Grid, Row and Cells and we can further tweak this in a follow up issue. 

Reverting this and bringing back `UserInputSettings` here will have Two 'Key metrics' section which is not good.

Reply:
> the `UserInputSettings` will be removed as part of #6210, and is only displayed in Dashboard (not in Settings).

I checked that ticket and don't see where it says that that component should be removed. Is that the right ticket?

> If I revert this change, then there will be multiple titles which is not desired ...

Which multiple titles do you mean? This ticket is about adding a new toggle to the admin settings that controls whether to display key metrics on the dashboard page. Which multiple titles we will see in that toggle?

> ... and does not satisfy the ticket

Where in the ticket it says that we need to remove that component? I see IB says `Update assets/js/components/settings/SettingsAdmin.js to include SettingsKeyMetrics before UserInputPreview.`, but there is nothing else regarding that component.

> Reverting this and bringing back UserInputSettings here will have Two 'Key metrics' section which is not good.

I believe @jimmymadon knows about this and there is a ticket (or will be created a ticket) to resolve this. Am I right, Jimmy?

Reply:
> I checked that ticket and don't see where it says that that component should be removed. Is that the right ticket?

It doesn't specifically say that, yes. However, the way it will be redesigned in that issue will make the notification incompatible or overkill in its usage in `SettingsAdmin` because we will not need it in settings. See [here](https://www.figma.com/file/CdtrHB0HAjfGRWTnTMMQI0/Key-Metrics?type=design&node-id=1980-63901&t=ZHEhseYz59lKIEOb-0) for the design. 

> Where in the ticket it says that we need to remove that component? I see IB says `Update assets/js/components/settings/SettingsAdmin.js to include SettingsKeyMetrics before UserInputPreview.`, but there is nothing else regarding that component.

In the same design, you can see that the toggle should be displayed even if the user has not answered the user input questions, because user may have hand-picked the metrics. If we follow the IB and put the toggle before `UserInputPreview`, it will not be displayed when user has not answered the questions which is not correct. 

This is admittedly a bit hard to follow, but I think reverting the `SettingsAdmin` to `develop` and just adding the goggle before `UserInputPreview` is less correct than the current test, and we can definitely refine this in the future if needs be. However, if you still feel strongly in favor or a revert here, I'll do it. Just note that in that case, the toggle will not be displayed in all cases. 

Cheers.

Reply:
> However, the way it will be redesigned in that issue will make the notification incompatible or overkill in its usage in `SettingsAdmin` because we will not need it in settings.

Ok, then, could you please update that ticket to explicitly say that we need to remove `UserInputSettings` because otherwise it is just your assumption that it will be removed in that ticket and an engineer that will work on it may not know about it and will not delete that. Once that ticket IB is updated, I'll approve this ticket.

Reply:
Thanks, @kuasha420.